Description: Add -g -Wall to CFLAGS/CXXFLAGS.
Author: Miriam Ruiz <miriam@debian.org>
--- a/Sankore-3.1/Sankore_3.1.pro
+++ b/Sankore-3.1/Sankore_3.1.pro
@@ -394,8 +394,8 @@
     CONFIG += link_prl
     LIBS += -lcrypto
     LIBS += -lX11
-    QMAKE_CFLAGS += -fopenmp
-    QMAKE_CXXFLAGS += -fopenmp
+    QMAKE_CFLAGS += -fopenmp -g -Wall
+    QMAKE_CXXFLAGS += -fopenmp -g -Wall
     QMAKE_LFLAGS += -fopenmp
     UB_LIBRARY.path = $$DESTDIR
     UB_I18N.path = $$DESTDIR/i18n
