Description: Use hard-coded paths for Debian package.
Author: Miriam Ruiz <miriam@debian.org>
Index: sankore/Sankore-3.1/src/frameworks/UBPlatformUtils_linux.cpp
===================================================================
--- sankore.orig/Sankore-3.1/src/frameworks/UBPlatformUtils_linux.cpp	2013-03-07 20:08:11.603607252 +0100
+++ sankore/Sankore-3.1/src/frameworks/UBPlatformUtils_linux.cpp	2013-03-07 20:08:31.000000000 +0100
@@ -38,7 +38,7 @@
 
 QString UBPlatformUtils::applicationResourcesDirectory()
 {
-    return QApplication::applicationDirPath();
+    return QString("/usr/share/open-sankore/");
 }
 
 void UBPlatformUtils::hideFile(const QString &filePath)
Index: sankore/Sankore-3.1/src/web/UBWebController.cpp
===================================================================
--- sankore.orig/Sankore-3.1/src/web/UBWebController.cpp	2013-03-07 20:08:11.603607252 +0100
+++ sankore/Sankore-3.1/src/web/UBWebController.cpp	2013-03-07 20:08:11.595607255 +0100
@@ -192,11 +192,8 @@
 
     QString tutorialHtmlIndexFile = 0;
     QString tutorialPath = "/etc/Tutorial/tutorial" + language + "/index.html";
-#if defined(Q_WS_MAC)
-    tutorialHtmlIndexFile = QApplication::applicationDirPath()+ "/../Resources" + tutorialPath;
-#else
-    tutorialHtmlIndexFile = QApplication::applicationDirPath() + tutorialPath;
-#endif
+
+    tutorialHtmlIndexFile = "/usr/share/open-sankore/" + tutorialPath;
 
     QUrl currentUrl = QUrl::fromLocalFile(tutorialHtmlIndexFile);
 
