Description: Use hard-coded paths for Debian package.
Author: Miriam Ruiz <miriam@debian.org>
--- a/Sankore-3.1/src/frameworks/UBPlatformUtils_linux.cpp
+++ b/Sankore-3.1/src/frameworks/UBPlatformUtils_linux.cpp
@@ -39,7 +39,7 @@
 
 QString UBPlatformUtils::applicationResourcesDirectory()
 {
-    return QApplication::applicationDirPath();
+    return QString("/usr/share/open-sankore/");
 }
 
 void UBPlatformUtils::hideFile(const QString &filePath)
--- a/Sankore-3.1/src/web/UBWebController.cpp
+++ b/Sankore-3.1/src/web/UBWebController.cpp
@@ -193,11 +193,8 @@
 
     QString tutorialHtmlIndexFile = 0;
     QString tutorialPath = "/etc/Tutorial/tutorial" + language + "/index.html";
-#if defined(Q_WS_MAC)
-    tutorialHtmlIndexFile = QApplication::applicationDirPath()+ "/../Resources" + tutorialPath;
-#else
-    tutorialHtmlIndexFile = QApplication::applicationDirPath() + tutorialPath;
-#endif
+
+    tutorialHtmlIndexFile = "/usr/share/open-sankore/" + tutorialPath;
 
     QUrl currentUrl = QUrl::fromLocalFile(tutorialHtmlIndexFile);
 
