Description: Build against Qt 4.7.
Author: Miriam Ruiz <miriam@debian.org>
Index: sankore/Sankore-3.1/src/adaptors/UBMetadataDcSubsetAdaptor.cpp
===================================================================
--- sankore.orig/Sankore-3.1/src/adaptors/UBMetadataDcSubsetAdaptor.cpp	2013-03-07 20:08:11.667607255 +0100
+++ sankore/Sankore-3.1/src/adaptors/UBMetadataDcSubsetAdaptor.cpp	2013-03-07 20:08:11.635607254 +0100
@@ -115,7 +115,7 @@
     xmlWriter.writeTextElement(UBSettings::uniboardDocumentNamespaceUri, "size", QString("%1x%2").arg(width).arg(height));
 
     // introduced in UB 4.4
-    xmlWriter.writeTextElement(UBSettings::uniboardDocumentNamespaceUri, "updated-at", UBStringUtils::toUtcIsoDateTime(QDateTime::currentDateTimeUtc()));
+    xmlWriter.writeTextElement(UBSettings::uniboardDocumentNamespaceUri, "updated-at", UBStringUtils::toUtcIsoDateTime(QDateTime::currentDateTime().toUTC()));
     // introduced in OpenSankore 1.40.00
     xmlWriter.writeTextElement(UBSettings::uniboardDocumentNamespaceUri,UBSettings::sessionTitle,proxy->metaData(UBSettings::sessionTitle).toString());
     xmlWriter.writeTextElement(UBSettings::uniboardDocumentNamespaceUri,UBSettings::sessionAuthors,proxy->metaData(UBSettings::sessionAuthors).toString());
Index: sankore/Sankore-3.1/src/board/UBFeaturesController.cpp
===================================================================
--- sankore.orig/Sankore-3.1/src/board/UBFeaturesController.cpp	2013-03-07 20:08:11.667607255 +0100
+++ sankore/Sankore-3.1/src/board/UBFeaturesController.cpp	2013-03-07 20:08:11.647607253 +0100
@@ -41,6 +41,10 @@
 
 #include "gui/UBFeaturesWidget.h"
 
+#if QT_VERSION < 0x040700
+inline bool qHash(const QUrl &url) { return qHash(url.toString()); }
+#endif
+
 const QString UBFeaturesController::virtualRootName = "root";
 const QString UBFeaturesController::rootPath  = "/" + virtualRootName;
 
Index: sankore/Sankore-3.1/src/gui/UBTeacherGuideWidgetsTools.cpp
===================================================================
--- sankore.orig/Sankore-3.1/src/gui/UBTeacherGuideWidgetsTools.cpp	2013-03-07 20:08:11.667607255 +0100
+++ sankore/Sankore-3.1/src/gui/UBTeacherGuideWidgetsTools.cpp	2013-03-07 20:08:11.651607252 +0100
@@ -625,11 +625,15 @@
     setLayout(mpLayout);
     mpTitle = new QLineEdit(this);
     mpTitle->setObjectName("UBTGLineEdit");
+#if QT_VERSION >= 0x040700
     mpTitle->setPlaceholderText(tr("Insert link title here..."));
+#endif
     mpUrl = new QLineEdit(this);
     connect(mpUrl,SIGNAL(editingFinished()),this,SLOT(onUrlEditionFinished()));
     mpUrl->setObjectName("UBTGLineEdit");
+#if QT_VERSION >= 0x040700
     mpUrl->setPlaceholderText("http://");
+#endif
     mpLayout->addWidget(mpTitle);
     mpLayout->addWidget(mpUrl);
 }
Index: sankore/Sankore-3.1/src/domain/UBGraphicsWidgetItem.cpp
===================================================================
--- sankore.orig/Sankore-3.1/src/domain/UBGraphicsWidgetItem.cpp	2013-03-07 20:08:11.667607255 +0100
+++ sankore/Sankore-3.1/src/domain/UBGraphicsWidgetItem.cpp	2013-03-07 20:08:11.663607255 +0100
@@ -79,7 +79,9 @@
     page()->setNetworkAccessManager(UBNetworkAccessManager::defaultAccessManager());
 
     setAcceptDrops(true);
+    #if QT_VERSION >= 0x040700
     setAutoFillBackground(false);
+    #endif
 
     QPalette pagePalette = page()->palette();
     pagePalette.setBrush(QPalette::Base, QBrush(Qt::transparent));
