Description: Fix FTBFS when built with gcc 4.7.
Author: Miriam Ruiz <miriam@debian.org>
Index: sankore/Sankore-3.1/src/frameworks/UBPlatformUtils_linux.cpp
===================================================================
--- sankore.orig/Sankore-3.1/src/frameworks/UBPlatformUtils_linux.cpp	2013-03-07 20:08:11.711607255 +0100
+++ sankore/Sankore-3.1/src/frameworks/UBPlatformUtils_linux.cpp	2013-03-07 20:08:11.699607252 +0100
@@ -28,6 +28,8 @@
 #include <X11/Xlib.h>
 #include <X11/keysym.h>
 
+#include <unistd.h>
+
 #include "frameworks/UBFileSystemUtils.h"
 
 
Index: sankore/Sankore-ThirdParty/trolltech/singleapplication/qtlocalpeer.cpp
===================================================================
--- sankore.orig/Sankore-ThirdParty/trolltech/singleapplication/qtlocalpeer.cpp	2013-03-07 20:08:11.711607255 +0100
+++ sankore/Sankore-ThirdParty/trolltech/singleapplication/qtlocalpeer.cpp	2013-03-07 20:08:11.703607252 +0100
@@ -50,6 +50,8 @@
 #endif
 #if defined(Q_OS_UNIX)
 #include <time.h>
+#include <unistd.h>
+#include <sys/types.h>
 #endif
 
 namespace QtLP_Private {
